iocore/net/I_NetVConnection.h

```diff
@@ -823,6 +834,8 @@ class NetVConnection : public AnnotatedVConnection
   bool is_transparent;
   /// Set if proxy protocol is enabled
   bool is_proxy_protocol;
+  /// This is a tri-state, -1 == no MPTCP on listening port, 0 == no MPTCP negotiate, 1 == MPTCP negotiated
+  int mptcp_state;
```

Comment:
Personally I would go with 0, 1, 2 and not deal with a signed type.  Also, this could be a smaller type and match the type in the httpsm (which really should be a smaller type).

Reply:
I was contemplating that, but this felt more normal. Having “1” mean no MPTCP negotiates seemed odd.

I don’t care a tremendous amount , but -1 felt that it fit it with a lot of things we do elsewhere.